deb_files/postrm

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
```

Comment:
Same, why use `set -e` if defaulting all commands to `|| true` if they fail?
Would make sense to remove `|| true` and let `set -e` do its job, or add proper error handling for commands.